docs/source/conf.py

```diff
@@ -88,7 +88,7 @@
 # further.  For a list of options available for each theme, see the
 # documentation.
 html_theme_options = {
-    'description': "Elyra extends JupyterLab Notebooks with an AI centric approach",
+#    'description': "Elyra extends JupyterLab Notebooks with an AI centric approach",
```

Comment:
Curious to what is the issue here?

Reply:
'description' isn't a valid html_theme option and the following is produced:
```
preparing documents... WARNING: unsupported theme option 'description' given
```

Reply:
@lresende - I left it commented out in case there's another place it should go, but would prefer it be removed.

Reply:
We can remove it.